### DISCUSSION TITLE
How does autohotkey work in cs2?

### DISCUSSION THREAD

**Discussion by DerErzengel** [McDaived/NoRecoil-CS2#17] [General]
i came across this project because i am trying to create my own autohotkey script. But unfortunately cs2 does not seem to recognize the input. How do you handle this? 

thanks for your answer!

**Comment by Synx400** [comment]
Bro everything is okay on this Ahk but the recoil pattern is not fully accurate and its not smooth it's easy to detect that your not legit player when someone observe you on the game..

**Reply by DerErzengel** [reply]
That's not my point. I'm trying to figure out how it works at all that c2 accepts the input from this tool here, because since cs2 came out, the developers seem to be trying to disable ahk scripts or input from scripts.

**Comment by muslera602** [comment]
Hi
Volume 2 is having a problem and I can't lower it, but I can lower Volume 1.
But only the controls for the AK47, Galil, and Famas work. The rest don't activate during the game. What could this be?
